dashboard/app/models/pd/workshop.rb

```diff
@@ -74,6 +74,19 @@ class Pd::Workshop < ActiveRecord::Base
     ]
   }
 
+  COURSE_SHORT_NAME_MAP = {
+    COURSE_CSF => 'csf',
+    COURSE_CSP => 'csp',
+    COURSE_ECS => 'ecs',
+    COURSE_CS_IN_A => 'CSinA',
+    COURSE_CS_IN_S => 'CSinS',
```

Comment:
Because `csina` seems weird. I could do `cs_in_a`, or make all of them capitalized, e.g. `CSF`. Preference?